src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimCallbackHandler.java

```diff
@@ -151,15 +162,31 @@ private CallbackResponse populateRespondentCopyObjects(CallbackParams callbackPa
             isRespondent1 = NO;
         }
 
+        DynamicList courtLocationList = courtLocationUtils.getLocationsFromList(fetchLocationData(callbackParams));
```

Comment:
this line can be moved inside the if (V1), so that it is not loaded if not needed. At least the actual call, since the value is going to be used both for r1 and r2

Reply:
good spot, i'll change that

Reply:
actually after second thought, i think it's ok to be in there, the V1 bits would be removed almost immediately.
it'll almost be doing extra work to remove it later.